Makefile

```diff
@@ -50,25 +50,28 @@ binary: build
 	$(DOCKER_RUN_DOCKER) hack/make.sh binary
 
 cross: build
-	$(DOCKER_RUN_DOCKER) hack/make.sh binary cross
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary binary cross
 
 deb: build
-	$(DOCKER_RUN_DOCKER) hack/make.sh binary build-deb
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary build-deb
+
+dynbinary: build
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary
```

Comment:
Yeah just for testing its nice






On Thu, Nov 12, 2015 at 7:47 AM -0800, "Tianon Gravi" <notifications@github.com> wrote:












In Makefile:
>  
>  deb: build
> -	$(DOCKER_RUN_DOCKER) hack/make.sh binary build-deb
> +	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary build-deb
> +
> +dynbinary: build
> +	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary

If you want to build a binary for your host from within the Docker
container, the only sure solution is the static binary.



—
Reply to this email directly or view it on GitHub.